packages/integration-tests/cypress.json

```diff
@@ -27,8 +27,8 @@
     "REMOTE_SSH_ID_FILE": "",
     "REMOTE_USER": "",
     "REMOTE_HOST": "",
-    "PULL_SECRET": "",
-    "API_BASE_URL": "",
+    "PULL_SECRET": "{\"auths\":{\"cloud.openshift.com\":{\"auth\":\"==\",\"email\":\"some-user@redhat.com\"},\"quay.io\":{\"auth\":\"==\",\"email\":\"some-user@redhat.com\"},\"registry.connect.redhat.com\":{\"auth\":\"\",\"email\":\"some-user@redhat.com\"},\"registry.redhat.io\":{\"auth\":\"\",\"email\":\"some-user@redhat.com\"}}}",
```

Comment:
The pull-secret must be identified as valid in the UI, otherwise the "Next" button in Create flow doesn't become enabled.